### ISSUE TITLE
CMake on Windows complain that I need VS 2017 15.5, while I am using VS 2017 15.5

### ISSUE THREAD

**Issue by benapetr** [SFTtech/openage#968]
I have VS 2017 15.5.5, this is what I get when I try to run CMake:

```
C:\Users\Petr Bena\Documents\openage\build>cmake -DCMAKE_TOOLCHAIN_FILE=D:\libs\vcpkg\scripts\buildsystems\vcpkg.cmake -G "Visual Studio 15 2017 Win64" ..


 ___  ______ _______ _______ ___
|  _)/ _____|_______|_______|_  |
| | ( (____  _____      _     | |    ___  ____  _____ ____  _____  ____ _____
| |  \____ \|  ___)    | |    | |   / _ \|  _ \| ___ |  _ \(____ |/ _  | ___ |
| |_ _____) ) |        | |   _| |  | |_| | |_| | ____| | | / ___ ( (_| | ____|
|___|______/|_|        |_|  (___|   \___/|  __/|_____)_| |_\_____|\___ |_____)
                                         |_|                     (_____|

Welcome to the SFT technologies computer-aided openage build system!

You have chosen, or been chosen, to attempt the daring task of building openage.
If you have installed all the dependencies that are conveniently listed in
[doc/building.md], this _might_ just work!

If it doesn't, consider reporting the issue: https://github.com/SFTtech/openage
Or ask for help:
  * Matrix: #sfttech:matrix.org
  * IRC:    #sfttech on freenode.net

-- Found Windows SDK 10.0.16299.0
-- The CXX compiler identification is MSVC 19.12.25835.0
-- Check for working CXX compiler: D:/Microsoft Visual Studio/2017/Community/VC/Tools/MSVC/14.12.25827/bin/Hostx86/x64/cl.exe
-- Check for working CXX compiler: D:/Microsoft Visual Studio/2017/Community/VC/Tools/MSVC/14.12.25827/bin/Hostx86/x64/cl.exe -- works
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Performing Test HAVE_THREAD_LOCAL_STORAGE
-- Performing Test HAVE_THREAD_LOCAL_STORAGE - Success
-- Performing Test HAVE_REQUIRED_CXX17_SUPPORT
-- Performing Test HAVE_REQUIRED_CXX17_SUPPORT - Failed


The compiler doesn't support required C++17 features:
  * Nested namespace definition (N4230)
  * Structured Bindings (P0217R3)
The following versions support these features:
  * clang++ >= 5
  * g++ >= 7
  * Microsoft Visual Studio 2017 >= 15.5
Please upgrade your compiler to build openage.


CMake Error at buildsystem/CheckCompilerFeatures.cmake:38 (message):
  aborting
Call Stack (most recent call first):
  CMakeLists.txt:132 (include)


-- Configuring incomplete, errors occurred!
See also "C:/Users/Petr Bena/Documents/openage/build/CMakeFiles/CMakeOutput.log".
See also "C:/Users/Petr Bena/Documents/openage/build/CMakeFiles/CMakeError.log".
```

I don't know how is this check for C++17 features executed, but my compiler support these features, so the check has some bug? Or maybe wrong compiler parameters are used? According to docs you should use this switch for it to work `/std:c++17`. Is it even there?

**Comment by TheJJ** [comment]
Can you confirm that #969 fixed this problem?

**Comment by TheJJ** [comment]
Closed through #969.
